cmd/kubeadm/app/apis/kubeadm/v1alpha1/zz_generated.defaults.go

```diff
@@ -28,15 +28,5 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
-	scheme.AddTypeDefaultingFunc(&MasterConfiguration{}, func(obj interface{}) { SetObjectDefaults_MasterConfiguration(obj.(*MasterConfiguration)) })
```

Comment:
Please don't touch the generated code.
I'm pretty sure this has to be here in order for kubeadm to function